selfdrive/car/ecu_addrs.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+import capnp
+import time
+import traceback
+from typing import Dict, Set
+
+import cereal.messaging as messaging
+from panda.python.uds import SERVICE_TYPE
+from selfdrive.car import make_can_msg
+from selfdrive.boardd.boardd import can_list_to_can_capnp
+from selfdrive.swaglog import cloudlog
+
+TESTER_PRESENT_DAT = bytes([0x02, SERVICE_TYPE.TESTER_PRESENT, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0])
```

Comment:
The first byte should be the subaddr here, correct?

Reply:
yes, but using them might complicate things quite a bit, and if it is only for toyota camera and radar maybe try counting that response addr as two matches for toyota first.